Makefile

```
@@ -76,3 +81,7 @@ web/run: ## Run web application while proxying backend requests
 web/run: web/install-deps
 	cd web \
 		&& yarn start:with-proxy
+
+web/set-yarn-version: ## Set yarn version
```

Nit: could use a `.PHONY` here too.

```suggestion
.PHONY: web/set-yarn-version
web/set-yarn-version: ## Set yarn version
```

---

Good catch!